core/services/ocr2/plugins/ocr2vrf/coordinator/ocr_cache.go

```diff
@@ -0,0 +1,120 @@
+package coordinator
+
+import (
+	"runtime"
+	"sync"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// ocrCache is a caching strucuture that allows items to be stored and then evicted
+// based on an eviction window. In this package, it is being used to track in-flight
+// items the coordinator includes in OCR reports, such that the items can be checked against
+// the cache to avoid double-transmissions.
+type ocrCache[T any] struct {
+	evictionWindow time.Duration
+	cacheMu        sync.Mutex
+	cache          map[common.Hash]*cacheItem[T]
+	cleaner        *intervalCacheCleaner[T]
+}
+
+type cacheItem[T any] struct {
+	item      T
+	itemKey   common.Hash
+	timestamp time.Duration
```

Comment:
`time.Duration` is not a timestamp - that's `time.Time`.
```suggestion
	timestamp time.Time
```